JDI.Light/JDI.Light/Elements/Complex/RadioButtons.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using JDI.Light.Elements.Base;
+using JDI.Light.Elements.Common;
+using JDI.Light.Exceptions;
+using JDI.Light.Factories;
+using JDI.Light.Interfaces.Complex;
+using OpenQA.Selenium;
+
+namespace JDI.Light.Elements.Complex
+{
+    public class RadioButtons : UIElement, IRadioButtons
+    {
+        public By RadioButtonLocator { get; set; }
+
+        public By LabelLocator { get; set; }
+
+        private List<Label> Labels => FindElements(LabelLocator)
+            .Select(element => UIElementFactory.CreateInstance<Label>(LabelLocator, this, element)).ToList();
+
+        private List<UIElement> Radios => FindElements(RadioButtonLocator)
```

Comment:
we can use RadioButton class here

Reply:
I can't find it in solution. Is it implemented in some branch?

Reply:
@DmitrievDmitrii , please find it here:
https://github.com/jdi-testing/jdi-light-csharp/blob/master/JDI.Light/JDI.Light/Elements/Common/RadioButton.cs

Reply:
this is previous name of this class.
i renamed it and moved to complex namespace, according to java